py/desispec/io/photo.py

```diff
@@ -52,29 +52,16 @@ def gather_targetdirs(tileid, fiberassign_dir=None):
     fahdr = fits.getheader(fiberfile, ext=0)
 
     # Gather the targeting directories.
-    targetdirs = [fahdr['TARG']]
-    for moretarg in ['TARG2', 'TARG3', 'TARG4', 'TARG5', 'TARG6']:
-        if moretarg in fahdr:
-            targetdirs += [fahdr[moretarg]]            
-
-    # Any secondary targets or ToOs?
-    if 'SCND' in fahdr:
-        for morescnd in ['SCND', 'SCND2', 'SCND3', 'SCND4', 'SCND5', 'SCND6']:
-            if morescnd in fahdr and fahdr[morescnd].strip() != '-':
-                targetdirs += [fahdr[morescnd]]
-        
-    if 'TOO' in fahdr:
-        TOOfile = fahdr['TOO']
-        # can be a KPNO directory!
-        if 'DESIROOT' in TOOfile:
-            TOOfile = os.path.join(desi_root, TOOfile.replace('DESIROOT/', ''))
-        if TOOfile[:6] == '/data/': # fragile
-            TOOfile = os.path.join(desi_root, TOOfile.replace('/data/', ''))
-        if 'afternoon_planning' in TOOfile:
-            TOOfile = TOOfile.replace('afternoon_planning/surveyops', 'survey/ops/surveyops') # fragile!
```

Comment:
quick remark (I ve not checked the rest of the code): wouldn t you want to keep those kpno-related lines?